.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "trailingComma": "none",
```

Comment:
Prettier default values have changed since Prettier v1. I added a config to keep the previous default values in order to minimize the number of changes resolved the the `eslint --fix` command. If we want to always stick with the prettier default values, I can remove this file once the review is complete. Let me know if there is any preference.